pl/python_installation/instructions.md

```diff
@@ -2,23 +2,27 @@
 
-Django jest napisane w Pythonie. Musimy posiadać Pythona, by zrobić cokolwiek w Django. Zacznijmy od jego instalacji! Chcemy zainstalować Python 3.6, więc jeżeli posiadasz jakąkolwiek wcześniejszą wersję, będziesz musiała ją uaktualnić.
+Django został w napisany w Pythonie. Potrzebujemy Pythona, żeby móc zrobić cokolwiek w Django. Zacznijmy zatem od zainstalowania go! Chcemy, zainstalować najnowszą wersję Python 3, więc jeśli masz starszą wersje, należy ją uaktualnić. Jeśli masz już w wersji 3.4 lub nowszej, powinno być dobrze.
 
 <!--sec data-title="Install Python: Windows" data-id="python_windows" data-collapse=true ces-->
 
-Na początku sprawdź, czy twój komputer działa na 32-bitowej czy 64-bitowej wersji Windowsa, poprzez wciśnięcie kombinacji przycisków Windows + Pause/Break, które otworzą Właściwości systemu - sprawdź w linii "Typ systemu". Możesz ściągnąć Pythona dla Windowsa ze strony https://www.python.org/downloads/windows/. Kliknij w link "Latest Python 3 Release - Python x.x.x". Jeżeli twój komputer pracuje na **64-bitowej** wersji Windowsa, ściągnij **Windows x86-64 executable installer**. W innym wypadku ściągnij **Windows x86 executable installer**. Po ściągnięciu instalatora, powinnaś go uruchomić (klikając dwukrotnie w niego) i postępować według wyświetlanych instrukcji.
+Najpierw sprawdź, czy na komputerze działa wersja 32-bitowa lub 64-bitowa wersja systemu Windows, w linijce "Typ systemu" w oknie o informacjach o systemie. Aby przejść do tego okna wypróbuj jedną z następujących metod:
 
-Na jedną rzecz będziesz musiała uważać: Podczas instalacji zauważysz ekran oznaczony jako "Setup". Upewnij się, że zaznaczyłaś checkbox "Add Python 3.6 to PATH" oraz kliknęłaś "Install Now", tak jak tutaj:
+* Wciśnij klawisz Windows i klawisz Pause/Break w tym samym czasie
+* Otwórz panelu sterowania z menu systemu Windows, a następnie przejdź do system & zabezpieczenia, następnie System
+* Wciśnij przycisk Windows, a następnie przejdź do Settings >System >About
 
-![Nie zapomnij dodać Pythona do ścieżki systemowej (ang. Path)](../python_installation/images/python-installation-options.png)
+Możesz ściągnąć Pythona dla Windowsa ze strony https://www.python.org/downloads/windows/. Kliknij w link "Latest Python 3 Release - Python x.x.x". Jeżeli twój komputer pracuje na **64-bitowej** wersji Windowsa, ściągnij **Windows x86-64 executable installer**. W innym wypadku ściągnij **Windows x86 executable installer**. Po ściągnięciu instalatora, powinnaś go uruchomić (klikając dwukrotnie w niego) i postępować według wyświetlanych instrukcji.
+
+Jedna rzecz na którą trzeba uważać: Podczas instalacji zauważysz okno oznaczona jako "Setup". Upewnij się, że zaznaczyłeś opcję "Add Python 3.6 to PATH" lub ''Add Python to your environment variables" i kliknąłeś "Install Now", jak pokazano tutaj (może to wyglądać trochę inaczej, jeśli instalujesz inną wersję).
```

Comment:
Jedna rzecz na -> Jedna rzecz, na
oznaczona -> that relates to ticked checkboxes, please rewrite